lti_consumer/lti_1p3/tests/test_consumer.py

```diff
@@ -261,6 +262,143 @@ def test_set_invalid_presentation_claim(self):
         with self.assertRaises(ValueError):
             self.lti_consumer.set_launch_presentation_claim(document_target="invalid")
 
+    @ddt.data(
+        "group",
+        "course_offering",
+        "course_section",
+        "course_template"
+    )
+    def test_set_valid_context_claim(self, context_type):
+        """
+        Check if setting context claim data works
+        """
+        self._setup_lti_user()
+        self.lti_consumer.set_context_claim(
+            "context_id",
+            context_types=[context_type],
+            context_title="context_title",
+            context_label="context_label"
+        )
+
+        expected_context_type_vocab = LTI_1P3_CONTEXT_TYPE_MAP[context_type]
+        expected_claim_data = {
+            "id": "context_id",
+            "type": [expected_context_type_vocab],
+            "title": "context_title",
+            "label": "context_label",
+        }
+
+        self.assertEqual(
+            self.lti_consumer.lti_claim_context,
+            {
+                "https://purl.imsglobal.org/spec/lti/claim/context": expected_claim_data
+            }
+        )
+
+        # Prepare LTI message
+        launch_request = self._get_lti_message()
+
+        # Decode and verify message
+        decoded = self._decode_token(launch_request['id_token'])
+        self.assertIn(
+            "https://purl.imsglobal.org/spec/lti/claim/context",
+            decoded.keys()
+        )
+        self.assertEqual(
+            decoded["https://purl.imsglobal.org/spec/lti/claim/context"],
+            expected_claim_data
+        )
+
+    def test_set_context_claim_type_non_list(self):
```

Comment:
This test is failing and isn't valid anymore.

Reply:
Removed